packages/api/src/external/aws/app-config.ts

```diff
@@ -152,14 +184,26 @@ export async function isWebhookPongDisabledForCx(cxId: string): Promise<boolean>
 
 export async function isEpicEnabledForCx(cxId: string): Promise<boolean> {
   const cxIdsWithEpicEnabled = await getCxsWithEpicEnabled();
+  const globalEnabled = await isFeatureFlagEnabledStringArray("cxsWithEpicEnabled");
+  if (!globalEnabled) return false;
 
   return cxIdsWithEpicEnabled.length === 0 ? true : cxIdsWithEpicEnabled.some(i => i === cxId);
 }
 
+export async function isDemoAugEnabledForCx(cxId: string): Promise<boolean> {
+  const cxIdsWithDemoAugEnabled = await getCxsWitDemoAugEnabled();
+  const globalEnabled = await isFeatureFlagEnabledStringArray("cxsWithDemoAugEnabled");
+  if (!globalEnabled) return false;
+
+  return cxIdsWithDemoAugEnabled.length === 0
+    ? true
+    : cxIdsWithDemoAugEnabled.some(i => i === cxId);
```

Comment:
I see your comment about following the Epic approach (no cx means all cxs). I think this was a mistake w/ that one, we were thinking "this flag is temporary, when we're ready to have all cxs enabled for Epic we can just clean the list", but that's not the case, its part of the flow and we'll likely always have an item there.

I think it will be a long way until we decide that all cxs should have this enabled by default (inverting the FF), if ever. Even then, we'd need to update the code.

I'd take the approach that empty means no one is enabled for it. Safer during the release phase, and the alternative doesn't seem to help in any way. Thoughts?

Reply:
100% agreed -- the harder it is to turn it on for everyone, the less likely a mistake.

Reply:
Fixed.